src/Runner/RunnerConfig.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Runner;
+
+use PhpCsFixer\Runner\Parallel\ParallelConfig;
+
+/**
+ * @author Greg Korba <greg@codito.dev>
+ */
+final class RunnerConfig
```

Comment:
+`@internal` ?

Reply:
Yeah, I believe it's a leftover from initial implementation, where `RunnerConfig` was used in `Runner`'s constructor, so it had to be public just like runner. Now it can be internal.